From ae611eca18037147acad8df308f5a1a20255d996 Mon Sep 17 00:00:00 2001
From: Jean-Yves Avenard <jyavenard@mythtv.org>
Date: Tue, 15 Jul 2014 14:59:39 +1000
Subject: [PATCH] metadata: Fix broken retrieval for screenshots

Test could never have been evaluated to true

(cherry picked from commit a50a24b0f408957885ff928569a061df07e5e993)
---
 mythtv/libs/libmythmetadata/metadatafactory.cpp |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mythtv/libs/libmythmetadata/metadatafactory.cpp b/mythtv/libs/libmythmetadata/metadatafactory.cpp
index 0109213..d24c233 100644
--- a/mythtv/libs/libmythmetadata/metadatafactory.cpp
+++ b/mythtv/libs/libmythmetadata/metadatafactory.cpp
@@ -313,7 +313,7 @@ void MetadataFactory::OnSingleResult(MetadataLookup *lookup)
             map.insert(kArtworkBanner, info);
         }
 
-        if (!lookup->GetType() == kMetadataRecording)
+        if (lookup->GetType() != kMetadataRecording)
         {
             ArtworkList screenshotlist = lookup->GetArtwork(kArtworkScreenshot);
             if (screenshotlist.size())
-- 
1.7.10.2

